### DISCUSSION TITLE
1.0.0 Release Candidate

### DISCUSSION THREAD

**Discussion by ArchLeaders** [NX-Editor/.github#1] [Announcements]
### Todo:

* ~~Cleanup Singleton View/ViewModel ViewLocater implementation~~ (https://github.com/NX-Editor/NxEditor/commit/285079d0880a6b19bbeeb3704b0ec16223233f27)
* ~~Preload config modules in PluginLoader~~ (https://github.com/NX-Editor/NxEditor/commit/0d27ca82e045b71697cf45ffc66fde3845ba3852)
* Cleanup launcher ShellViewModel
* ~~Filter out Avalonia logs~~ (https://github.com/NX-Editor/NxEditor/commit/ecf03f75ed39942c17084bf9a10da01a5d0e0f38)
* ~~Implement Singleton ConfigView/ConfigPageModel View/ViewModel~~ (https://github.com/NX-Editor/NxEditor/commit/285079d0880a6b19bbeeb3704b0ec16223233f27)
* ~~Implement StatusModal in application status footer~~ (https://github.com/NX-Editor/NxEditor/commit/fd72ae51effdf10f9c4f03df0631054b2e05a65a)
* ~~Fix recent files menu collection~~
* ~~Cleanup pre-launch config validation~~

### Changes

* Module system rewrite (NxEditor.PluginBase)
  * NxEditor.EpdPlugin
  * NxEditor.TotkPlugin
  * NxEditor.PluginTemplate
* Avalonia 11 updates
* Common launcher/installer (NxEditor.Launcher)

**Comment by Waikuteru** [comment]
Looking good! Does the NxEditor.TotkPlugin include a RSTB table? The only thing I'm really missing so far is the RSTB table view before the RSTB change.

**Reply by ArchLeaders** [reply]
Yeah it will contain the string table for the RESTBL

**Reply by Waikuteru** [reply]
Awesome!

**Comment by ArchLeaders** [comment]
1.0.0-rc1 is actually out (finally)!

https://github.com/NX-Editor/NxEditor/releases/tag/1.0.0-rc1
